src/components/Accordion/Accordion.module.css

```diff
@@ -13,10 +13,149 @@
   list-style: none;
   border-bottom-style: solid;
   border-bottom-width: var(--eds-theme-border-width);
-  border-bottom-color: var(--eds-theme-color-neutral-subtle-border);
+  border-bottom-color: var(--eds-theme-color-border-neutral-subtle);
 }
 
+/**
+ * Inverted accordion
+ * 1) Make the text lighter to be accessible on darker backgrounds
+ */
 .accordion--inverted {
-  color: var(--eds-theme-color-inverted-text);
-  border-bottom-color: var(--eds-theme-color-neutral-subtle-border-inverted);
+  color: var(--eds-theme-color-text-neutral-default-inverse);
+  border-bottom-color: var(--eds-theme-color-border-neutral-strong);
+}
+
+/**
+ * Individual accordion panel
+ */
```

Comment:
currently, in the CODE_GUIDELINES, we have:

> Compound component children have an associated `.module.css` file, and child component styles will contain styles relevant to the subcomponent element (e.g. `AccordionPanel.module.css` would begin with `.accordion__panel { ... }`).

To be consistent, could we keep the accordion panel styles in `AccordionPanel.module.css`?

Reply:
That's fine. I saw `PrimaryNav` and `UtilityNav` not have that and I think I rolled with that. Happy to break that back out

Reply:
I can do a pass at all of the ones that are using the parent to make sure they're all consistent

Reply:
Took a pass at the ones I worked on. Should be good now I think